gateway-service/src/main/java/org/zowe/apiml/gateway/filters/pre/RequestContextProvider.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.gateway.filters.pre;
+
+import com.netflix.zuul.context.RequestContext;
+
+public interface RequestContextProvider {
+    RequestContext context();
```

Comment:
Depending on how the extenders require RequestContextProvider for v1 compatibility, it may also be best to add contextProvider back as part of the constructor for ServiceNotFoundFilter (so that we fully revert this specific RequestContextProvider change in commit ab3681). What exactly are the extenders using RequestContextProvider for?

Reply:
@AmandaDErrico I will verify with the extender, but I believe they just need this class to exist, not to be used anywhere.

Reply:
I verified with extender and this work for them.